functions/test/produce.test.js

```diff
@@ -0,0 +1,81 @@
+const expect = require("expect");
+const request = require("supertest");
+
+const app = require("../index");
+const test = require("firebase-functions-test")();
+const { produce } = require("./data");
+
+let id;
+
+describe("POST /produce", () => {
+  it("create a new produce", (done) => {
+    request(app)
+      .post("/produce")
+      .send(produce)
+      .expect((res) => {
```

Comment:
I'm going to keep pushing back to make these `request` and `error`. Maybe @Caffeinix has a different opinion, but I don't think we need to follow the Firebase docs on this one